Package.swift

```diff
@@ -241,6 +241,7 @@ let package = Package(
             dependencies: [
                 .target(name: "Apodini"),
                 .target(name: "ApodiniVaporSupport"),
+                .target(name: "ApodiniREST"),
```

Comment:
This dependency is also introduced with my renewed ExporterConfiguration, therefore fits perfectly fine 👍